.github/CODEOWNERS

```diff
@@ -17,6 +17,8 @@
 /plugins/techdocs                           @backstage/techdocs-core
 /plugins/techdocs-backend                   @backstage/techdocs-core
 /plugins/ilert                              @yacut
+/plugins/home                               @backstage/techdocs-core
+/packages/app/src/components/home           @backstage/techdocs-core
```

Comment:
In my opinion, this line about an example-app component feels like inflating CODEOWNERS tbh.

Reply:
Ok! Fixed in 4ea6e81